test/e2e/node/security_context.go

```diff
@@ -252,10 +252,11 @@ func testPodSELinuxLabeling(f *framework.Framework, hostIPC bool, hostPID bool)
 	pod.Spec.SecurityContext.SELinuxOptions = &v1.SELinuxOptions{
 		Level: "s0:c0,c1",
 	}
-
```

Comment:
nit: unrelated change.

Reply:
this *is* related. The `f.TestContainerOutput` call is related to the test case above, not below, but the vertical whitespace suggested otherwise, so I fixed it.

In other words, it was like that before:
```go
        pod.Spec.SecurityContext.SELinuxOptions = &v1.SELinuxOptions{
                Level: "s0:c0,c1",
        }

[1]     f.TestContainerOutput("Pod with same MCS label reading test file", pod, 0, []string{testContent})
[2]     // Confirm that the same pod with a different MCS
        // label cannot access the volume
```

In the above snippet, `[1]` marks the end of the previous test case, and `[2]` marks the beginning of a new test case.

After my changes, it looks like this:
```go
        pod.Spec.SecurityContext.SELinuxOptions = &v1.SELinuxOptions{
                Level: "s0:c0,c1",
        }
[1]     f.TestContainerOutput("Pod with same MCS label reading test file", pod, 0, []string{testContent})

[2]     // Confirm that the same pod with a different MCS
        // label cannot access the volume
```

Hope that makes sense

Reply:
@oomichi  ^^^